bigbluebutton-html5/imports/ui/components/nav-bar/settings-dropdown/component.jsx

```diff
@@ -17,6 +14,37 @@ import DropdownList from '/imports/ui/components/dropdown/list/component';
 import DropdownListItem from '/imports/ui/components/dropdown/list/item/component';
 import DropdownListSeparator from '/imports/ui/components/dropdown/list/separator/component';
 
+const intlMessages = defineMessages({
+  optionsLabel: {
+    id: 'app.dropdown.optionsLabel',
```

Comment:
The namespace should be `app.settings.dropdown` or `app.settingsDropdown` instead of `app.dropdown`